### ISSUE TITLE
No such file or directory 'block_list.txt'

### ISSUE THREAD

**Issue by kilabyte** [pinheadmz/ClockBlocker#1]
When loading "ledbits" this is the error i get - "No block history file found. - block_list.txt". I have the node running and synced 100% and the RPC user and password has been set. Any ideas?

**Comment by pinheadmz** [comment]
Did you add the `notify` settings in `bitcoin.conf`? Check the readme, bitcoind needs to be configured to edit the block list file.

**Comment by kilabyte** [comment]
yes both walletnotify and blocknotify were added as per the readme

**Comment by pinheadmz** [comment]
You may need to create a `data` directory and set the permissions to `777`.
Or create these files manually before running: https://github.com/pinheadmz/ClockBlocker/blob/henners-not-adafruit/ledbits.py#L38-L40
